geometry_msgs/Pose pose
geometry_msgs/Twist velocity
float64 distance
float64 major_semiaxis
float64 minor_semiaxis
